llvm/lib/MC/MCParser/AsmParser.cpp

```diff
@@ -3733,6 +3737,19 @@ bool AsmParser::parseDirectiveLoc() {
   return false;
 }
 
+/// parseDirectiveLoc
+/// ::= .loc_label label
+bool AsmParser::parseDirectiveLocLabel(SMLoc DirectiveLoc) {
+  StringRef Name;
+  DirectiveLoc = Lexer.getLoc();
+  if (parseIdentifier(Name))
+    return TokError("expected identifier");
```

Comment:
no parsing error test. `rg ERR=1 llvm/test/MC/ELF` for some examples.

Reply:
Added error test for syntax errors and duplicate label.